packages/apps/job-launcher/server/src/common/enums/webhook.ts

```diff
@@ -1,7 +1,7 @@
 export enum EventType {
   ESCROW_CREATED = 'escrow_created',
   ESCROW_CANCELED = 'escrow_canceled',
-  TASK_CREATION_FAILED = 'task_creation_failed',
+  ESCROW_FAILED = 'escrow_failed',
```

Comment:
Can both be added? Cvat oracles are working with `TASK_CREATION_FAILED` so we'll keep it until they move to the new `ESCROW_FAILED`
```suggestion
  TASK_CREATION_FAILED = 'task_creation_failed',
  ESCROW_FAILED = 'escrow_failed',
```

Reply:
added